### ISSUE TITLE
Need to handle zero probability case for DRO

### ISSUE THREAD

**Issue by gbyeon** [Argonne-National-Laboratory/DSP#182]
For DRO test cases, there can be a scenario with zero `probability`. In the following code, we may need to prevent `obj_` from becoming `Inf`:
https://github.com/Argonne-National-Laboratory/DSP/blob/2c53e5aaeff5dbb6c5a9f25fd9080a4eea7911b6/src/Solver/DualDecomp/DdSub.cpp#L250-L251

**Comment by gbyeon** [comment]
For DRO test cases, there can be a scenario with zero `probability`. To prevent `obj_` from becoming `Inf`, we may need to modify the code as follows?
```
if (probability > 1e-6)
		{
			for (int j = 0; j < tssModel->getNumCols(1); ++j)
				obj_[tssModel->getNumCols(0)+j] /= probability;
		}
```

**Comment by gbyeon** [comment]
Need to modify 
https://github.com/Argonne-National-Laboratory/DSP/blob/2c53e5aaeff5dbb6c5a9f25fd9080a4eea7911b6/src/Solver/DualDecomp/DdDroWorkerUB.cpp#L92
and
https://github.com/Argonne-National-Laboratory/DSP/blob/2c53e5aaeff5dbb6c5a9f25fd9080a4eea7911b6/src/Model/StoModel.cpp#L1372-L1378

**Comment by kibaekkim** [comment]
Resolved by the commit above and PR #197
